setup.py

```diff
@@ -94,6 +94,7 @@ def _get_readme():
                         'pexpect>=3.3,<4.0',
                         'requests>=2.4.1,<3.0',
                         'pytest>=2.6.2,<3.0',
+                        'pytest-catchlog>=1.2.2,<2.0',
```

Comment:
Is this just making sure the version of catchlog is what we're expecting? And if so isn't 1.2.2 the latest version (so why less than 2.0? Is it just in case they break things with updates?)